book/src/volume-health-monitor.md

```diff
@@ -2,37 +2,32 @@
 
 ## Status
 
-Status | CSI external-health-monitor-controller sidecar Version | CSI external-health-monitor-agent sidecar Version
---|--|--
-Alpha | 0.1.0 | 0.1.0
+Status | Min K8s Version | Max K8s Version | external-health-monitor-controller Version
+--|--|--|--
+Alpha | 1.21 | - | 0.3.0
```

Comment:
Can you list the latest external-health-monitor-controller Version which is 0.8.0?

Reply:
Done